puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_develop.yaml

```diff
@@ -0,0 +1,41 @@
+- job:
+    name: test_develop
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 32
+    properties:
+      - github_foreman
+    scm:
+      - foreman
+    triggers:
+      - github_daily
+    axes:
+      - axis:
+         type: user-defined
+         name: ruby
+         values:
+           - 1.9.3
+           - 2.0.0
+           - 2.1
+      - axis:
+          type: user-defined
+          name: database
+          values:
+            - postgresql
+            - sqlite3
+            - mysql
+    builders:
+      - test_develop
+    publishers:
+      - gemset_cleanup
+      - ircbot_freenode
+      - archive:
+          artifacts: 'pkg/*,Gemfile.lock'
+          only-successful: false
```

Comment:
Change this to `only-if-success` now, the newer JJB had a different name for the setting to our local change.